internal/serve/httphandler/disbursement_handler_test.go

```diff
@@ -142,20 +145,36 @@ func Test_DisbursementHandler_PostDisbursement(t *testing.T) {
 		{
 		   "name": "My New Disbursement name 5",
 		   "wallet_id": "aab4a4a9-2493-4f37-9741-01d5bd31d68b",
-		   "asset_id": "61dbfa89-943a-413c-b862-a2177384d321"
+		   "asset_id": "61dbfa89-943a-413c-b862-a2177384d321",
+		   "verification_type": "date_of_birth"
 		}`
 
 		want := `{"error":"Request invalid", "extras": {"country_code": "country_code is required"}}`
 
 		assertPOSTResponse(t, ctx, handler, method, url, requestBody, want, http.StatusBadRequest)
 	})
 
-	t.Run("returns error when wallet_id is not valid", func(t *testing.T) {
+	t.Run("returns error when no verification type is provided", func(t *testing.T) {
 		requestBody, err := json.Marshal(PostDisbursementRequest{
 			Name:        "disbursement 1",
 			CountryCode: country.Code,
 			AssetID:     asset.ID,
-			WalletID:    "aab4a4a9-2493-4f37-9741-01d5bd31d68b",
+			WalletID:    enabledWallet.ID,
+		})
+		require.NoError(t, err)
+
+		want := `{"error":"Verification field invalid", "extras": {"verification_type": "invalid parameter. valid values are: DATE_OF_BIRTH, PIN, NATIONAL_ID_NUMBER"}}`
```

Comment:
We should either go with `verification_type` or `verification field`. there is a mix of both in this error message.